src/main/webapp/app/scores/exercise-scores.component.ts

```diff
@@ -34,9 +36,12 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
     // make constants available to html for comparison
     FilterProp = FilterProp;
 
-    readonly QUIZ = ExerciseType.QUIZ;
-    readonly PROGRAMMING = ExerciseType.PROGRAMMING;
+    readonly FILE_UPLOAD = ExerciseType.FILE_UPLOAD;
     readonly MODELING = ExerciseType.MODELING;
+    readonly PROGRAMMING = ExerciseType.PROGRAMMING;
```

Comment:
you can also just use 
`readonly ExerciseType = ExerciseType;`

then in the html, you use ExerciseType.PROGRAMMING or ExerciseType.MODELING

Reply:
True, that's also how it is done in the participation component. I refactored it.